public/index.html

```diff
@@ -1 +1,12 @@
-<div id="root"></div>
\ No newline at end of file
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>metastudent.se</title>
+    <link rel="icon" href="favicon.png">
+</head>
+<body>
+    <div id="root"></div>
+</body>
+</html>
```

Comment:
Please end all files with an empty line, per convention (yes I know it was already wrong, but you might as well fix it)